src/utils/env-utils.ts

```diff
@@ -20,6 +20,8 @@ declare global {
 const ENABLED = 'true';
 
 export const isProduction = import.meta.env.PROD;
+export const defaultDataPlaneSuffix = import.meta.env
+    .VITE_DEFAULT_DATA_PLANE_SUFFIX;
```

Comment:
This is risky... so if this is not set this would fail silently. Unless you know of a way where this could support being an empty string.

I don't like all the `getXYZ` in that we need to keep calling functions that we shouldn't... but here I think we can just write a iife that sets this and then throws an error if it isn't there.